Interesting. The same things seems to be happening on Travis CI: it timed
out after 10 minutes.

But on my local, this test immediately fails due to an R dataframe access
error:

# 

ERROR: test_two_genes_nonzero_clr_two_conditions_positive_influence

## (inferelator_ng.tests.test_bbsr_R.TestDR)

Traceback (most recent call last):
  File
"/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/tests/test_bbsr_R.py",
line 132, in test_two_genes_nonzero_clr_two_conditions_positive_influence
    (betas, resc) = self.brd.run(self.X, self.Y, self.clr, self.priors)
  File "/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/bbsr_R.py", line
100, in run
    utils.call_R(driver_path)
  File "/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/utils.py", line
33, in call_R
    return subprocess.check_output(command, shell=True)
  File "//anaconda/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command 'R -f
/Users/ndeveaux/Dev/inferelator_ng/inferelator_ng/tests/artifacts/run_bbsr.R'
returned non-zero exit status 1

On Wed, Jul 6, 2016 at 9:23 AM, Aaron Watters notifications@github.com
wrote:

> Hmmm... I'm running the tests locally and I've got 2 R processes maxing
> out the CPU for a couple minutes now. Maybe we can find out more about this
> at 10am.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/simonsfoundation/inferelator_ng/pull/14#issuecomment-230769717,
> or mute the thread
> https://github.com/notifications/unsubscribe/AAGbtphjStiEIDj8cp6ewuSWFHLhIQvBks5qS6xqgaJpZM4JF7S0
> .

--- a/inferelator_ng/tests/test_bbsr_R.py
+++ b/inferelator_ng/tests/test_bbsr_R.py
@@ -126,8 +126,8 @@ class TestDR(unittest.TestCase):
 
     def test_two_genes_nonzero_clr_two_conditions_positive_influence(self):
         self.set_all_zero_priors()
-        self.X = pd.DataFrame([[1, 1], [2, 2]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
-        self.Y = pd.DataFrame([[1, 1], [2, 2]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
+        self.X = pd.DataFrame([[1, 2], [1, 2]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
+        self.Y = pd.DataFrame([[1, 2], [1, 2]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
         self.clr = pd.DataFrame([[.1, .1],[.1, .2]], index = ['gene1', 'gene2'], columns = ['gene1', 'gene2'])
         (betas, resc) = self.brd.run(self.X, self.Y, self.clr, self.priors)
         self.assert_matrix_is_square(2, betas)
@@ -137,8 +137,8 @@ class TestDR(unittest.TestCase):
 
     def test_two_genes_nonzero_clr_two_conditions_zero_gene1_positive_influence(self):
         self.set_all_zero_priors()
-        self.X = pd.DataFrame([[0, 0], [2, 2]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
-        self.Y = pd.DataFrame([[0, 0], [1, 1]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
+        self.X = pd.DataFrame([[0, 2], [0, 2]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
+        self.Y = pd.DataFrame([[1, 2], [1, 2]], index = ['gene1', 'gene2'], columns = ['ss1', 'ss2'])
         self.clr = pd.DataFrame([[.1, .1],[.1, .2]], index = ['gene1', 'gene2'], columns = ['gene1', 'gene2'])
         (betas, resc) = self.brd.run(self.X, self.Y, self.clr, self.priors)
         self.assert_matrix_is_square(2, betas)

